holidays/countries/dominican_republic.py

```diff
@@ -31,9 +31,9 @@ class DominicanRepublic(HolidayBase):
     @staticmethod
     def __change_day_by_law(holiday, latest_days=(THU, FRI)):
         # Law No. 139-97 - Holidays Dominican Republic - Jun 27, 1997
-        if holiday >= date(1997, 6, 27):
+        if holiday >= date(1997, JUN, 27):
             if holiday.weekday() in {TUE, WED}:
-                holiday -= rd(weekday=MO(-1))
+                holiday += rd(weekday=MO(-1))
             elif holiday.weekday() in latest_days:
                 holiday += rd(weekday=MO(1))
```

Comment:
Not sure if this'd be more consistent with other use cases.
```suggestion
                holiday += rd(weekday=MO(+1))
```